lms/djangoapps/bulk_email/signals.py

```diff
@@ -5,10 +5,13 @@
 from django.dispatch import receiver
 
 from student.models import CourseEnrollment
+import logging
```

Comment:
move it on top

Reply:
would it be causing any sort of performance adjustment?

Reply:
reason behind asking you to move them up (to group them properly) was

> Imports should be grouped in the following order:
> 
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
it was orphan and it is removed 🥇